src/Tests/UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -2221,24 +2221,32 @@ static Object MakeObject()
 
             static void TestImports()
             {
-                var (initializeWithWindow, windowNative) = MakeImports();
-                
-                GC.Collect();
-                GC.WaitForPendingFinalizers();
+                var (initializeWithWindow, windowNative) = MakeImports();
+
+                GCCollect();
 
                 var hwnd = new IntPtr(0x12345678);
                 initializeWithWindow.Initialize(hwnd);
                 Assert.Equal(windowNative.WindowHandle, hwnd);
             }
 
+            static void GCCollect()

```

Comment:
Style here is still evolving but I would recommend putting all local functions at the end of the function. Placing them in the middle makes reading the function logical flow cumbersome.